modules/nw-egress-router-about.adoc

```diff
@@ -0,0 +1,117 @@
+requires fewer changes to the client pods to get them to work. Programs can be
+told to use an HTTP proxy by setting an environment variable.
+
+In _DNS proxy mode_, the egress router runs as a DNS proxy for TCP-based
+services from its own IP address to one or more destination IP addresses. Client
+pods that want to make use of the reserved, source IP address must be modified
+to connect to the egress router rather than connecting directly to the
+destination IP. This ensures that external destinations treat traffic as though
+it were coming from a known source.
+
+Redirect mode works for all services except for HTTP and HTTPS. For HTTP and
+HTTPS services, use HTTP proxy mode. For TCP-based services with IP addresses
+or domain names, use DNS proxy mode.
+
+[id="nw-egress-router-about-deployments_{context}"]
+== Deployment considerations
+
+The egress router adds a second IP address and MAC address to the primary
+network interface of the node.
+As a result, you might need to configure your hypervisor or cloud provider to allow the additional address.
+
+Red Hat OpenStack Platform::
+
+If you are deploying {product-title} on Red Hat OpenStack Platform, you must
+whitelist the IP and MAC addresses on your OpenStack environment, otherwise
+link:https://access.redhat.com/solutions/2803331[communication will fail]:
++
+----
+neutron port-update $neutron_port_uuid \
+  --allowed_address_pairs list=true \
+  type=dict mac_address=<mac_address>,ip_address=<ip_address>
+----
+
+Red Hat Enterprise Virtualization::
+
+If you are using
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.2/html/Administration_Guide/Red_Hat_Enterprise_Virtualization_Manager_configuration_options_explanations_limitations_and_best_practices.html[Red
+Hat Enterprise Virtualization], you must set
+`EnableMACAntiSpoofingFilterRules` to `false`.
+
+VMware vSphere::
+
+If you are using VMware vSphere, see the link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.security.doc/GUID-3507432E-AFEA-4B6B-B404-17A020575358.html[VMWare documentation for securing vSphere standard switches]. View and change VMWare vSphere default settings by selecting the host's virtual switch from the vSphere Web Client.
+
+Specifically, ensure that the following are enabled:
+
+* https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.security.doc/GUID-942BD3AA-731B-4A05-8196-66F2B4BF1ACB.html[MAC Address Changes]
+* https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.security.doc/GUID-7DC6486F-5400-44DF-8A62-6273798A2F80.html[Forged Transits]
+* https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.security.doc/GUID-92F3AB1F-B4C5-4F25-A010-8820D7250350.html[Promiscuous Mode Operation]
+
+[id="nw-egress-router-about-failover_{context}"]
+== Failover configuration
+
+To avoid downtown, you can deploy an egress router with a ReplicationController resource.
```

Comment:
Probably at this point it's a better idea to use a Deployment instead of a ReplicationController, I think the only change in the yaml would be setting `kind: Deployment`. Haven't tested it though.

Also we tell the user to access it from a service, so ideally the documentation should also include the service, or at least tell the user to do `oc expose deployment/<deployment name>`